strings.conf

```diff
@@ -5,6 +5,7 @@
 #counters
 !counter 0x1162 Bunny Ears Counter
 !counter 0x163 Counter(Gunkan Suship Chef's Choice)
+!counter 0x65 Signal Counter
```

Comment:
Signal Counter is `0x104d` and is used by _Signal Warrior_ and _On Your Mark, Get Set, DUEL!_ already

Reply:
haii